src/pages/category/Index.vue

```diff
@@ -1,43 +1,86 @@
-        <a-tree
-          :expanded-keys="expandedKeys"
-          :auto-expand-parent="autoExpandParent"
-          :show-line="true"
-          :tree-data="categories"
-          :replaceFields="{children:'children', title:'name', key:'id' }"
-          @expand="onExpand"
+        <a-card title="分类列表" style="height:calc(100vh - 164px)">
+          <a slot="extra" href="#">新增</a>
+          <a-tree
+            :tree-data="categories"
+            :replaceFields="{children:'children', title:'name', key:'id' }"
+            autoExpandParent
+          >
+          </a-tree>
+        </a-card>
+      </a-col>
+      <a-col :span="12" style="margin-top:50px;">
+        <a-form
+          :form="editModel"
+          :label-col="{ span: 5 }"
+          :wrapper-col="{ span: 12 }"
+          @submit="createOrUpdateCategory"
         >
-          <template slot="title" slot-scope="{ title }">
-            <span v-if="title.indexOf(searchValue) > -1">
-              {{ title.substr(0, title.indexOf(searchValue)) }}
-              <span
-                style="color: #f50"
-              >{{ searchValue }}</span>
-              {{ title.substr(title.indexOf(searchValue) + searchValue.length) }}
-            </span>
-            <span v-else>{{ title }}</span>
-          </template>
-        </a-tree>
+          <a-form-item label="分类名称">
+            <a-input v-model="editModel.name" />
+          </a-form-item>
+          <a-form-item label="上级分类">
+            <a-select v-model="editModel.parentCategoryId" placeholder="无上级分类" allowClear>
+              <a-select-option
+                v-for="category in allCategories"
+                :key="category.id"
+                :value="category.id"
+              >{{ category.name }}</a-select-option>
+            </a-select>
+          </a-form-item>
+          <a-form-item label="排序">
+            <a-input-number id="inputNumber" v-model="editModel.priority" :min="1" :max="1000000" />
+          </a-form-item>
+          <a-form-item label="分类描述">
+            <a-textarea
+              placeholder
+              v-model="editModel.description"
+              :auto-size="{ minRows: 3, maxRows: 6 }"
+            />
+          </a-form-item>
+          <a-form-item :wrapper-col="{ span: 12, offset: 5 }">
+            <a-button type="primary" html-type="submit">提交</a-button>
+          </a-form-item>
+        </a-form>
       </a-col>
-      <a-col :span="12">col-12</a-col>
     </a-row>
   </div>
 </template>
 
 <script>
-import httpClient from "../../core/services/httpclient";
-
 export default {
   data() {
     return {
       expandedKeys: [],
       searchValue: "",
       autoExpandParent: true,
-      categories: []
+      allCategories: [],
+      categories: [],
+      editModel: {
+        id: null,
+        name: "",
+        description: "",
+        priority: 99,
+        parentCategoryId: ""
+      },
+      rules: {
```

Comment:
for the clean component design, validation rules should be a definitely an independent section of component and DON'T going to dirty the `data`.
because they (validation rules) are actually NOT a part of real data.

You can try to check what [vuelidate](https://vuelidate.js.org/) has doing for a reference.

Reply:
@baiyunchen according to our conversation in WeChat, we could ignore this comment now and just follow your decision.